.travis/deploy_key.pem

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQD1SymHWWk75BntVfeAL1XJCO+glT4SXIqkWCRHsvuYRc++5VdqAOqcGzbzhzL2p2bRe+OML0HmxlQ793cbaiAgJNHusjCP5W1y/woe0+rpLxhWaMmderuqOmLOeh9/lbPWw61FpZRob9tqsn8v3fo1hVOpJets6N1CvGICR2xOZCOlELLakk38mKvifRQWccNF641D2PboGlJ2oNSL5LwKkzYwX1gReZwWIeBvMZpiOSLnWBAf+0SnaAtqUPATeuse8m4Fv6kTyqpWgoJsm1VRnJDDOFOqouyXd6CKtlpabCpQe3mhbiJVLFe10gLawUoL4uSRWiBWlbTlLSw4xOnZ boisbakhodirov@BOISs-MacBook-Pro.local
```

Comment:
Is this your personal unencrypted ssh key or the encrypted Travis version?

Reply:
I'm trying to hide. I was testing with travis ci. I'm not sure how secure it should be. I'm gonna put it in .gitignore